daemon.go

```diff
@@ -59,12 +60,27 @@ func (daemon menderDaemon) StopDaemon() {
 
 func (daemon *menderDaemon) Run() error {
 	// figure out the state
-	switch daemon.GetState() {
-	case MenderFreshInstall:
-		//do nothing
-	case MenderRunningWithFreshUpdate:
-		daemon.CommitUpdate()
+	for {
```

Comment:
Not sure if we need a loop here. At least for now, all the state transitions are killing the daemon (which is another thing to discuss).